angular/projects/public-nrpti/src/app/records/records-list/search-filters/search-filters.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="search-filter-container">
+  <section>
+    <div class="filter-header">
+      <span><h3>Filter</h3></span>
```

Comment:
Not sure why you need a span here, also mostly bootstrap classes should be sufficient.

Reply:
Removed